packages/dd-trace/src/appsec/rasp.js

```diff
@@ -53,6 +133,21 @@ function handleResult (actions, req) {
       config.appsec.stackTrace.maxStackTraces
     )
   }
+
+  const blockingAction = getBlockingAction(actions)
+  if (blockingAction && abortController) {
+    const rootSpan = web.root(req)
+    // Should block only in express
+    if (rootSpan.context()._name === 'express.request' && !abortOnUncaughtException) {
```

Comment:
I could suggest something like rootSpan?.context?.()?._name

```suggestion
    if (rootSpan?.context()._name === 'express.request' && !abortOnUncaughtException) {
```